src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -200,16 +200,6 @@ public static String formatDate(Date date) {
         sdf.setTimeZone(TimeZone.getTimeZone("UTC"));
         return sdf.format(date);
     }
-
-    /**
-     * Formats a date in the format dd MMM yyyy, HH:mm. Example: 05 May 2012,
-     * 22:04<br />
-     */
-    public static String formatTime(Date date) {
-        if (date == null)
-            return "";
-        return new SimpleDateFormat("EEE, dd MMM yyyy, HH:mm").format(date);
```

Comment:
It is not adjusted at all. It simply stays as what the instructor inputs.
Comments are different because we have to figure out the time the comment is posted, but for feedback sessions, we can just take it as the instructor gives. (along with the specified timezone)